Upgrade validators - systemd (#245)
moondance-labs/tanssi-docs: node-operators/validators/operational-tasks/index.md

---
title: Operational Tasks - Validators
description: Learn how to manage your activities as a validator in the Tanssi network protocol, including managing your account, upgrading your node, and more.
icon: octicons-arrow-switch-24
template: index-page.html
---

# Operational Tasks

> Keeping your validator up and running requires regular maintenance to ensure consistent performance and secure management of your accounts. Regular updates are essential to keep your node compatible with the latest Tanssi protocol changes, while tools such as proxy accounts provide an extra layer of security by allowing you to delegate certain tasks without exposing your primary account credentials. This section provides the necessary steps to keep your operations smooth, reliable, and secure.

Reviewer: ```suggestion
Keeping your validator running requires regular maintenance to ensure consistent performance and secure account management. Regular updates are essential to keep your node compatible with the latest Tanssi protocol changes. At the same time, tools such as proxy accounts provide an extra layer of security by allowing you to delegate specific tasks without exposing your primary account credentials. This section provides the steps to keep your operations smooth, reliable, and secure.
```
